shared/src/test/diff/ucs/Wildcard.mls

```diff
@@ -0,0 +1,54 @@
+:NewParser
+:NewDefs
+
+type Option[T] = None | Some[T]
+module None
+class Some[T](value: T)
+//│ type Option[T] = Some[T] | None
+//│ module None()
+//│ class Some[T](value: T)
+
+type Either[A, B] = Left[A] | Right[B]
+class Left[A](leftValue: A)
+class Right[B](rightValue: B)
+//│ type Either[A, B] = Left[A] | Right[B]
+//│ class Left[A](leftValue: A)
+//│ class Right[B](rightValue: B)
+
+// FIXME
+fun w1(x, e_0, e_1) =
+  if x is
+    Left(None) then 1
+    Right(None) then 2
+    _ and e_0 is y_0 and x is
+      Left(Some(lv)) then 3
+      _ and e_1 is y_1 and x is
+        Right(Some(rv)) then 4
+//│ ╔══[WARNING] Found duplicated branch
+//│ ╟── This decision path tries to fit
```

Comment:
Note: the "tries to fit" wording is confusing. I don't understand it.

Reply:
Updated. How do you think of this?

```
// It should report duplicated branches.
:w
fun testF(x) = if x is
  Foo(a) then a
  Foo(a) then a
//│ ╔══[WARNING] Found a duplicated branch
//│ ╟── This branch
//│ ║  l.21: 	  Foo(a) then a
//│ ║        	              ^
//│ ╟── is subsumed by the branch here.
//│ ║  l.20: 	  Foo(a) then a
//│ ╙──      	              ^
//│ testF: (Foo & {x: 'x}) -> 'x
//│      = [Function: testF]
```

Reply:
Oh, that's much clearer! 👍